src/interfaces/IGreeter.sol

```diff
@@ -65,9 +55,18 @@ interface IGreeter {
    */
   function token() external view returns (IERC20 _token);
 
+  /*///////////////////////////////////////////////////////////////
+                            LOGIC
+  //////////////////////////////////////////////////////////////*/
+  /**
+   * @notice Sets a new greeting
+   * @dev Only callable by the owner
+   * @param _newGreeting The new greeting to be set
+   */
+  function setGreeting(string memory _newGreeting) external;
```

Comment:
Regarding the reorder, I think the linting warning keeps showing due to the `VARIABLES` section above. 🤷‍♂️